.github/workflows/lint-ecosystem-order.yml

```diff
@@ -30,5 +30,19 @@ jobs:
         with:
           script: |
             const script = require('./.github/scripts/lint-ecosystem.js')
-            await script({ core })
+            const result = await script({ core })
```

Comment:
Can we handle everything in the script instead of writing chunks of code inside this yaml?

Reply:
Can we yes, should we 🤷

Reply:
(pushing update to move inside script)